server/api/region.go

```diff
@@ -308,10 +308,10 @@ func (h *regionsHandler) GetAll(w http.ResponseWriter, r *http.Request) {
 }
 
 // @Tags region
-// @Summary List regions start from a key.
-// @Param key query string true "Region start key"
-// @Param endkey query string true "Range end key"
-// @Param limit query integer false "Limit count" default(16)
+// @Summary List regions in a given range[startKey,endKey).
+// @Param startKey query string true "Region range start key"
```

Comment:
hi, @disksing , should I use `key` here?

Reply:
oh I think so. The document should match the code.